### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by gfhghfghfh** [TurtleShip/webtoon-crawler#5]
https://comic.naver.com//list.nhn?titleId=89097&page=1

stop 85%

**Comment by TurtleShip** [comment]
@gfhghfghfh  Thanks for the report.
Frankly, I haven't been able to update this repo for a while now, so it may take some time until I get back to this.

Is this problem reproducible for other webtoons, or just this one?

**Comment by gfhghfghfh** [comment]
Yes,when I  download some comics which do not have dynamic effects , the some folders are empty.
besides,when I download comics which have dynamic effects  , the all  folders are empty.

**Comment by gfhghfghfh** [comment]
comics  have dynamic effects
example
https://comic.naver.com/webtoon/list.nhn?titleId=714568&weekday=wed
https://comic.naver.com/webtoon/list.nhn?titleId=666670&weekday=tue
